ikarus/finiteelements/mechanics/linearelastic.hh

```diff
@@ -151,6 +154,10 @@ namespace Ikarus {
       return [&]([[maybe_unused]] auto gp) { return materialTangent(); };
     }
 
+    std::shared_ptr<const Geometry> geometry() const { return geo_; }
```

Comment:
This method should not return the shared_ptr. It should just share the reference. use `*geo`